bucket/cormanlisp.json

```diff
@@ -0,0 +1,43 @@
+{
+    "version": "3.1.2b",
+    "description": "A Common Lisp development environment integrated with Windows.",
+    "homepage": "https://github.com/sharplispers/cormanlisp",
+    "license": "MIT",
+    "suggest": {
+        "vcredist": "extras/vcredist2015"
+    },
+    "url": "https://github.com/sharplispers/cormanlisp/releases/download/v3.1.2b/CormanLisp-3.1.2b.zip",
+    "hash": "8bf722bd919a69ff743e2ae6448fcb3ccbc8be8b3ef746e6acc5cdcb1853ea95",
+    "extract_dir": "Corman Lisp",
+    "bin": [
+        "clconsole.exe",
+        "clconsoleapp.exe"
+    ],
+    "env_set": {
+        "CORMANLISP_HOME": "$dir"
+    },
+    "shortcuts": [
+        [
+            "CormanLisp.exe",
+            "Corman Lisp IDE"
+        ],
+        [
+            "documentation\\CormanLisp.pdf",
+            "Corman Lisp Manual"
+        ],
+        [
+            "clconsole.exe",
+            "Corman Lisp REPL",
+            "",
+            "CormanLisp.exe"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/sharplispers/cormanlisp",
+        "regex": "tree/v([\\w.]+)/"
+    },
+    "autoupdate": {
+        "url": "https://github.com/sharplispers/cormanlisp/releases/download/v$version/CormanLisp-$version.zip"
+    },
+    "notes": "Please restart your command line for CORMANLISP_HOME to take effect. Consider installing <https://www.quicklisp.org/>."
```

Comment:
> Consider installing <https://www.quicklisp.org/>

Why?

Reply:
This is copied from the SBCL notes. QuickLisp is the primary library manager for Common Lisp (think pip/PyPI or Cargo/crates.io).